### PR TITLE
Visual tweaks

### PR DESCRIPTION
**Pull request by terotic**
- Visual (mainly spacing and typography) style fixes
- Some small screen optimisation
- Add unit on numeric berth details

### CODE REVIEW

**Comment by midori0507** [comment]
@terotic Can you fix merge conflicts ?

File: src/components/forms/sections/OverviewInfo.tsx [hunk 1]
```diff
@@ -105,17 +105,17 @@ const OverviewInfo = ({
...
             <LocalizedLink to={steps[1].linkTo} className="vene-overview-info__edit-link">
-              <Icon name="pencil" />
               <FormattedMessage tagName="span" id="page.overview.info.edit" />
+              <Icon name="pencil" />
             </LocalizedLink>
-          </Col>
-        </Row>
+          </div>
+        </div>
         <Row>
```
Comment (midori0507) [inline src/components/forms/sections/OverviewInfo.tsx]: Did you forget to remove `Row` in here too?

File: src/components/forms/sections/OverviewInfo.tsx [hunk 2]
```diff
@@ -59,17 +59,17 @@ const OverviewInfo = ({
       <Container fluid>
         {!isWinterStorageFormMode && <OldBerthInfo application={application} />}
 
-        <Row>
-          <Col xs={8} md={10} className="vene-overview-info__header">
+        <div className="vene-overview-info__header">
```
Comment (midori0507) [inline src/components/forms/sections/OverviewInfo.tsx]: May i ask why ?

File: src/components/pages/berthPage/BerthPage.scss [hunk 3]
```diff
@@ -0,0 +1,11 @@
+@import 'helsinki/colors';
+
+.vene-berth-page {
+  &__tabs {
+    background-color: $hel-white;
+  }
+}
+
+.berth-filters-section {
```
Comment (midori0507) [inline src/components/pages/berthPage/BerthPage.scss]: bad name. `vene-berth-blabla`

File: src/components/pages/berthPage/BerthPage.tsx [hunk 4]
```diff
@@ -148,7 +148,7 @@ class BerthPage extends Component<Props> {
             }
           ]}
         />
-        <KoroSection color="fog" top>
+        <KoroSection color="fog" top className="berth-filters-section">
```
Comment (midori0507) [inline src/components/pages/berthPage/BerthPage.tsx]: `vene-koro-berth...`

File: src/components/forms/sections/OverviewInfo.tsx (hunk 2 above)
Comment (terotic) [reply src/components/forms/sections/OverviewInfo.tsx]: This is not really a place for grid for various reasons. Header and edit link are not really column-like elements in the layout. Also we want border-bottom throughout the row and applying it on the Row element would have lead to applying other styling and spacing on grid Row which it's not really ment for.

Comment (terotic) [reply src/components/forms/sections/OverviewInfo.tsx]: Maybe we can wrap the header element on full-width column to keep consistency with the grid markup and other Rows in the overview...

**Pull request merged**